src/EnergyPlus/DaylightingManager.cc

```diff
@@ -5579,13 +5579,7 @@ namespace DaylightingManager {
                     }
                 }
                 if (!daylightControlFound) {
-                    ShowSevereError("DaylightingDevice:Tubular = " + TDDPipe(PipeNum).Name + ":  is not connected to a Zone that has Daylighting.  ");
-                    ShowContinueError("Add Daylighting:Controls one of the following zones in the same enclosure:  ");
-                    for (int const zoneNum2 : DataViewFactorInformation::ZoneSolarInfo(Surface(SurfNum).SolarEnclSurfIndex).ZoneNums) {
-                        ShowContinueError(Zone(zoneNum2).Name);
-                    }
-                    ErrorsFound = true;
-                    CheckTDDZone(Surface(SurfNum).Zone) = false;
```

Comment:
`CheckTDDZone` is no longer relevant (in fact I don't see that it was used even in the current code), so delete it everywhere. And with `CheckTDDZone` gone, `CheckTDDs_firstTime` can also go away.

Reply:
I went through and removed all of these instances.